mhep/mhep/v1/migrations/0001_initial.py

```diff
@@ -1,28 +1,59 @@
-# Generated by Django 2.2.4 on 2019-08-15 15:20
+# Generated by Django 2.2.4 on 2019-10-24 14:31
 
+from django.conf import settings
 import django.contrib.postgres.fields.jsonb
 from django.db import migrations, models
+import django.db.models.deletion
+import mhep.v1.validators
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='Organisation',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.TextField()),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('members', models.ManyToManyField(blank=True, related_name='organisations', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Library',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.TextField()),
+                ('type', models.TextField()),
+                ('data', django.contrib.postgres.fields.jsonb.JSONField(default=dict, validators=[mhep.v1.validators.validate_dict])),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('owner', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='libraries', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'verbose_name_plural': 'libraries',
+            },
+        ),
         migrations.CreateModel(
             name='Assessment',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.TextField()),
-                ('description', models.TextField()),
+                ('description', models.TextField(blank=True)),
                 ('openbem_version', models.CharField(choices=[('10.1.0', 'v10.1.0'), ('10.1.1', 'v10.1.1')], max_length=20)),
-                ('status', models.CharField(choices=[('complete', 'Complete'), ('inprogress', 'In progress'), ('test', 'Test')], default='inprogress', max_length=20)),
-                ('data', django.contrib.postgres.fields.jsonb.JSONField()),
+                ('status', models.CharField(choices=[('Complete', 'Complete'), ('In progress', 'In progress'), ('Test', 'Test')], default='In progress', max_length=20)),
```

Comment:
Regenerating these has changed the `choices` key's from lower case, one word, into capitalized, spaced strings. Does that matter?

Reply:
Good spot, but it's actually remained the same - on `assessments` there were numerous migrations and a later one changed it to `In progress`. Because I deleted all the migrations and regenerated this one `initial` migration contains all the previous ones.

Reply:
👍